Doc/library/decimal.rst

```diff
@@ -1033,18 +1033,18 @@ New contexts can also be created using the :class:`Context` constructor
 described below. In addition, the module provides three pre-made contexts:
 
 
-.. class:: BasicContext
+.. data:: BasicContext
 
-   This is a standard context defined by the General Decimal Arithmetic
-   Specification.  Precision is set to nine.  Rounding is set to
+   This is a standard context defined by the General Decimal Arithmetic Specification.
```

Comment:
Please revert this. Reflowing the line length is not needed in this PR.

Reply:
Done

Reply:
Results show no more instances of :class: for BasicContext, ExtendedContext, or DefaultContext in the documentation.

Reply:
Thanks!